DOCS-2179: Generic gateways (#1193)
MultiSafepay/docs: content/integrations/plugins/magento2/faq/generic-gateways.md

---
title : "What is a generic gateway?"
meta_title: "Magento 2 plugin FAQ - Generic Gateway - MultiSafepay Docs"
meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
layout: "faqdetail"
read_more: "."
---

> The Magento 2 plugin offers a generic gateway. This is a gateway which you can customize yourself.

Reviewer: ```suggestion
The Magento 2 plugin offers a [generic gateway](/faq/general/generic-gateways/).
```

It can be found under the MultiSafepay payment methods inside the MultiSafepay configuration:

_Stores_ → _Configuration_ → _MultiSafepay_ → _Payment Methods_ → _Generic Gateway_

> You enter the gateway code and upload a custom gateway image.

Reviewer: ```suggestion
- You enter the gateway code and upload a custom gateway image.
```

> You also need to set if the gateway requires a shopping cart or not.

Reviewer: ```suggestion
- You also need to set if the gateway requires a shopping cart or not.
```

> It works for the most part like a regular redirect payment method.

Reviewer: ```suggestion
- It works for the most part as a regular redirect payment method.
```

> Except for recurring payments, all the features like refunding and creating a backend order are supported.

Reviewer: ```suggestion
- Except for recurring payments, all the features like refunding and creating a backend order are supported.
```

> If you have any questions regarding the integration of the generic gateway, please contact our integration team at integration@multisafepay.com

Reviewer: ```suggestion
- If you have any questions regarding the integration of the generic gateway, please contact our integration team at integration@multisafepay.com
```
